core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -410,13 +413,15 @@ public Response scheduleAsyncPersist(@QueryParam("path") final String path) {
    */
   @POST
   @Path(SET_ATTRIBUTE)
-  @ReturnType("java.lang.Void")
   public Response setAttribute(@QueryParam("path") final String path,
-      @QueryParam("pinned") final Boolean pinned, @QueryParam("ttl") final Long ttl,
-      @QueryParam("persisted") final Boolean persisted, @QueryParam("owner") final String owner,
-      @QueryParam("group") final String group, @QueryParam("permission") final Short permission,
-      @QueryParam("recursive") final Boolean recursive,
-      @QueryParam("ttxAction") final TtlAction ttlAction) {
+                               @QueryParam("pinned") final Boolean pinned,
+                               @QueryParam("ttl") final Long ttl,
+                               @QueryParam("persisted") final Boolean persisted,
+                               @QueryParam("owner") final String owner,
+                               @QueryParam("group") final String group,
+                               @QueryParam("permission") final Short permission,
+                               @QueryParam("recursive") final Boolean recursive,
+                               @QueryParam("ttxAction") final TtlAction ttlAction) {
```

Comment:
Could you revert this formatting change?

Reply:
this change fixes checkstyle error when building